docs/src/modules/pageDemo.js

```diff
@@ -262,6 +262,47 @@ class PageDemo extends React.PureComponent {
             ];
         }
 
+        const statsColumns = [
+            {
+                accessor: () => 'Super Cool Info Bar - Color: 11',
+                fontSize: 'large',
+                fontWeight: 'semibold',
+                header: null,
+                style: {
+                    marginBottom: '11px',
+                },
+                width: '100%',
+            }, {
+                accessor: (d) => (<div>Chart</div>), // eslint-disable-line no-unused-vars,max-len
+                fontWeight: 'bold',
+                header: null,
+            }, {
+                accessor: 'activeTemplates',
+                header: 'Active Templates',
+            }, {
+                accessor: 'inactiveTemplates',
+                header: 'Inactive Templates',
+            },
+        ];
+
+        const statsExtraColumns = [
+            {
+                columns: [
+                    {
+                        accessor: 'activeTemplates',
+                        header: 'Active Templates',
+                    }, {
+                        accessor: 'inactiveTemplates',
+                        header: 'Inactive Templates',
+                    },
+                ],
+                divide: true,
+            },
+        ];
```

Comment:
For demo purposes could we give the extra columns different names and different counts.  e.g. "Foo Templates" and "Bar Templates" maybe?  So we can tell that potentially different data is in the collapsible area?